clippy_config/src/conf.rs

```diff
@@ -39,6 +39,7 @@ const DEFAULT_DOC_VALID_IDENTS: &[&str] = &[
 ];
 const DEFAULT_DISALLOWED_NAMES: &[&str] = &["foo", "baz", "quux"];
 const DEFAULT_ALLOWED_IDENTS_BELOW_MIN_CHARS: &[&str] = &["i", "j", "x", "y", "z", "w", "n"];
+const DEFAULT_ALLOWED_PREPOSITIONS: &[&str] = &["to", "as", "into", "from"];
```

Comment:
I think it would be a good idea to let users extend this default configuration, check the `deserialize` function in line 665 if you want to add that in :D

Reply:
Good idea - implemented!